content/source/docs/cloud/run/modes-and-options.html.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: "cloud"
+page_title: "Run Modes and Options - Runs - Terraform Cloud and Terraform Enterprise"
+---
+
+# Run Modes and Options
+
+Terraform Cloud support many of the same modes and options available in local operations mode.
```

Comment:
(Starting to try and shy away from 'local operations mode')

```suggestion
Terraform Cloud runs support many of the same modes and options available in Terraform CLI.
```

Reply:
:fire-shh: